packages/functional-tests/pages/products/index.ts

```diff
@@ -7,16 +9,50 @@ export class SubscribePage extends BaseLayout {
 
   async setCreditCardInfo() {
     const frame = this.page.frame({ url: /elements-inner-card/ });
+    await frame.fill('.InputElement[name=cardnumber]', '');
```

Comment:
Q: Why set an empty cardnumber before filling it again below on L13?

Reply:
This is because I used this same function for the third test "subscribe with CC after initial failed subscription" and after entering an incorrect cc number the test was failing on the step to replace the CC number with correct details. So I had to delete the previous CC details(only cc number) and then re-enter it.